gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Install.groovy

```diff
@@ -6,6 +6,6 @@ class Install extends AdbTask {
 
     @TaskAction
     void exec() {
-        assertDeviceAndRunCommand(['install', '-r', apkPath])
+        assertDeviceAndRunCommand(['install', '-r', '-d', apkPath])
```

Comment:
See the commit message :-) My commit message usually contain valuable information, so you should not rely on reading just the combined diff of all commits.